util/settings/settings.go

```diff
@@ -118,6 +124,8 @@ const (
 	settingsWebhookBitbucketUUIDKey = "webhook.bitbucket.uuid"
 	// settingsApplicationInstanceLabelKey is the key to configure injected app instance label key
 	settingsApplicationInstanceLabelKey = "application.instanceLabelKey"
+	// resourcesCustomizationsKey is the key to the map of resource overrides
+	resourcesCustomizationsKey = "resource_customizations"
```

Comment:
I know it won't be consistent with our project directory name, but i think the configmap key should use dot notation to be consistent with the other settings keys (i.e. `resource.customizations`). It leaves room for having some other `resource.XXXXX` later.

Reply:
fixed